sway-lib-std/src/constants.sw

```diff
@@ -18,7 +19,9 @@ use ::contract_id::ContractId;
 ///     assert(BASE_ASSET_ID == msg_asset_id());
 /// }
 /// ```
-pub const BASE_ASSET_ID = ZERO_B256;
+pub const BASE_ASSET_ID: AssetId = AssetId {
```

Comment:
Should we move this into contract_id.sw (or wherever AssetId is defined)? That way, constants.sw doesn't need to depend on any other files.

Reply:
It used to depend on contract_id.sw before aswell, i dont see why this is a problem

Reply:
It's a good idea to have a file for constants. It's a safeguard for future extensions.
We have 2 constants atm, and I believe we've had more before (plus https://github.com/FuelLabs/sway/pull/4955#discussion_r1309778115), so I don't believe that we should be introducing another breaking change which moves it to another file. It already changes the type which has a meaningful point while moving it to an incorrect location does not